tests/python_package_test/test_dask.py

```diff
@@ -44,6 +44,7 @@
 CLIENT_CLOSE_TIMEOUT = 120
 
 tasks = ['binary-classification', 'multiclass-classification', 'regression', 'ranking']
+distributed_training_algorithms = ['data_parallel', 'voting_parallel']
```

Comment:
All 4 of these strings are valid for this argument (https://lightgbm.readthedocs.io/en/latest/Parameters.html#tree_learner):

* `data`
* `data_parallel`
* `voting`
* `voting_parallel`

But I don't think it's worth it to have all four choices here because that would create a lot of new tests (`2 * len(tasks) * len(data_output)`).

So could you do the following?

1. Change `distributed_training_algorithms` to the "main" values from https://lightgbm.readthedocs.io/en/latest/Parameters.html#tree_learner (`['data', 'voting_parallel']`)
2. add a new test like `test_training_respects_tree_learner_aliases()` that uses `['data_parallel', 'voting_parallel']`. It can just use `output="array"`, to reduce the total number of tests.

Reply:
looks like this was addressed in https://github.com/microsoft/LightGBM/pull/4088/commits/76b99941a8e8c016be9f375f09fe6d0af8e1108e